src/SolutionTemplate/UnoSolutionTemplate/UWP/Package.appxmanifest

```diff
@@ -29,7 +29,7 @@
 
   <Applications>
     <Application Id="App"
-      Executable="$ext_safeprojectname$.exe"
+      Executable="$targetnametoken$.exe"
```

Comment:
This will require a change in the transforms made to build the dotnet new templates.